docs/getting-started/installing-service-binding.md

```diff
@@ -6,11 +6,167 @@ sidebar_position: 1
 
 ## Compatiblity Note
 
-## Operator
+### Platforms
 
-_How can To be provided soon_
+The Service Binding Operator is compatible with:
+- Kubernetes >= v1.7.0
+- Openshift >= 4.5 
 
-## Helm
+### Pre-requisites
 
-_To be provided soon_
+- OLM : Follow the [installation guide](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/install/install.md) to install OLM, a tool to help manage operators running on the cluster.
+
```

Comment:
Should we add a mention that kubectl is needed?

Reply:
Yes. Done